src/components/pages/home/index.tsx

```diff
@@ -95,193 +105,223 @@ const Home: FunctionComponent<any> = ({homePageData}) => {
           />
         )}
       </div> */}
-      <div className="lg:space-y-6 space-y-4">
-        <Jumbotron resource={jumbotron} />
-        <section className="">
+      <div className="mt-12">
+        <WhatsNew resource={featureWhatsNew} />
+
+        <section className="mt-32">
+          <h2 className="md:text-xl text-lg sm:font-semibold font-bold mb-3 dark:text-white text-center">
+            Our Curated Guides
+          </h2>
```

Comment:
starting to add section titles and margin to space out the content on the homepage